src/dotnet/MLSearch.Service/Module/MLSearchSettings.cs

```diff
@@ -51,3 +52,8 @@ public sealed class OpenSearchSettings
     public string Password { get; set; } = "";
     public string ClientCertificatePath { get; set; } = "";
 }
+
+public sealed class EmbeddingsCalculatorSettings
+{
+    public string PredicationsUri { get; set; } = "http://localhost:28080/predictions/Alibaba-NLP_gte-multilingual-base";
```

Comment:
Is it applicable for staging and prod envs too?

Reply:
no